### PR TITLE
[Submission] Essay 

### PR DESCRIPTION
**Pull request by rymane**

### CODE REVIEW

**Comment by rymane** [comment]
Please note that in Overleaf the word count is 1959 and thus not exceeding the limit.

**Comment by wska** [comment]
Hi @rymane . I am just a fellow student (not a TA), but they have repeatedly mentioned that they will use a specific command for word count(which was changed very recently), which is:
> pdftotext \<file\> - | tr -d '.' | wc -w

This means that it DOES include things like references too. There has been lots of discussion surrounding the word count on the google doc.

If I were you, I would reach out to a TA or the professors and ask them if it is okay to change your essay slightly (by reducing its length) even after the deadline has passed. Otherwise, you are sure to receive a repeat on it, since they do not take into consideration other tools(like Overleaf) for word count. Only the specified command.

**Comment by khesoem** [comment]
Hi @rymane 

As @wska mentioned, your essay should not violate the word count limits.

@monperrus : Are students allowed to reduce the number of words now? @rymane's submission has 2170 words (70 words over the limit).

**Comment by monperrus** [comment]
@khaes-kth this was discussed at large during the lectures, I'm afraid there is no possible exception.

@rymane I'm merging this PR to acknowledge reception of the essay.

**Comment by rymane** [comment]
Hi @monperrus,

Thank you for your feedback. 

In the [grading criteria](https://github.com/KTH/devops-course/blob/2021/grading-criteria.md#essays), it is stated as a concept (and not a criteria) that the students should write a 2000-word essay, so I find it surprising that a strict 2000 word limit is now being enforced. If a ”concept” is argued by you to be considered a strict criteria, the grading criteria is very unclear, and any ambiguity by the school should not penalize a student’s grades. 

Even if there would be a strict word limit (which I argue there is not), my essay does not exceed 2000 words. It is generally accepted that the references, table of content etc., are not included in the word limit as they only facilitate the overview and, as such, are not part of the actual essay. It is, of course, fine to make exceptions from general rules, but any exceptions need to be clearly communicated and included in the instructions as well as the grading criteria. I appreciate that a strict limit might have been communicated at the lectures, but please keep in mind that the lectures are not mandatory and that, notwithstanding what has been communicated on voluntary lectures, the grading criteria remained unchanged.

I further kindly ask you to keep in mind that grading should be based on objective and transparent grounds. If I am considered to have breached the word limit, I cannot get a higher grade than E. I find it unreasonable that my grades should suffer for something that is not included in the grading criteria.

In summary, I kindly ask you to reconsider your decision. I simply ask that my essay should be graded objectively using the applicable grading criteria, as is the standard procedure and requirement on all courses on a university level in Sweden. 

I look forward to your response!

**Comment by monperrus** [comment]
We acknowledge your comment and answer over email. Best regards.
